packages/edge/android/src/main/java/com/adobe/marketing/mobile/reactnative/edge/RCTAEPEdgeArrayUtil.java

```diff
@@ -81,4 +82,31 @@ static WritableArray toWritableArray(Object[] array) {
 
         return writableArr;
     }
+
+    static WritableArray toWritableArray(ArrayList<?> array) {
+        if (array == null) {
+            return null;
+        }
+        WritableArray writableArr = Arguments.createArray();
+
+        for (Object value : array) {
+            if (value == null) {
+                writableArr.pushNull();
+            } else if (value instanceof Boolean) {
+                writableArr.pushBoolean((Boolean) value);
+            } else if (value instanceof Double) {
+                writableArr.pushDouble((Double) value);
+            } else if (value instanceof Integer) {
+                writableArr.pushInt((Integer) value);
+            } else if (value instanceof String) {
+                writableArr.pushString((String) value);
+            } else if (value instanceof Map) {
+                writableArr.pushMap(RCTAEPEdgeMapUtil.toWritableMap((Map<String, Object>) value));
```

Comment:
we should add the check for instanceof ArrayList to L105 and L78 too

Reply:
Tested with List and it also works.  Seems like Changing to List is better?